src/main/MQ2ImGuiConsole.cpp

```diff
@@ -64,6 +64,16 @@ static bool s_consoleVisible = false;
 static bool s_consoleVisibleOnStartup = false;
 static bool s_resetConsolePosition = false;
 static bool s_setFocus = false;
+static bool s_isConsoleLocked = false;
+static bool s_isTitleBarShown = false;
+static bool s_isMouseInFadeDelay = false;
+static bool s_updateConsoleFont = false;
+static int s_alphaConsoleNormal = 100;
+static int s_alphaConsoleFade = 100;
+static int s_consoleFontSize = 13;
+static auto s_mouseHoverTimerSecondsDelay = std::chrono::seconds(10);
+static std::chrono::steady_clock::time_point s_mouseHoverStartTimePoint;
+static std::string s_consoleFontName;
```

Comment:
I suspect a lot of these should be member variables. The reason why some of these exist as statics is because they hold state external to the console that is used to control it.

Most of this seems internal to the console and don't need to be here.